packages/wallet/src.ts/index.ts

```diff
@@ -220,6 +289,68 @@ export class Wallet extends Signer implements ExternallyOwnedAccount, TypedDataS
 		return encryptKeystore(this, password, options, progressCallback);
 	}
 
+	async call(txRequest : Deferrable<TransactionRequest>, blockTag?: BlockTag | Promise<BlockTag>): Promise<string> {
+		this._checkProvider("call");
+		const tx = await resolveProperties(this.checkTransaction(txRequest));
+		const contractAccountLikeID = getAccountFromAddress(tx.to.toString());
+		const contractId = `${contractAccountLikeID.shard}.${contractAccountLikeID.realm}.${contractAccountLikeID.num}`;
+
+		const thisAcc = getAccountFromAddress(await this.getAddress());
+		const thisAccId = `${thisAcc.shard}.${thisAcc.realm}.${thisAcc.num}`;
+
+		const nodeID = AccountId.fromString(tx.nodeId.toString());
+		const paymentTxId = TransactionId.generate(thisAccId);
+
+		const hederaTx = new ContractCallQuery()
+			.setContractId(contractId)
+			.setFunctionParameters(arrayify(tx.data))
+			.setNodeAccountIds([nodeID])
+			.setGas(Long.fromString(tx.gasLimit.toString()))
+			.setPaymentTransactionId(paymentTxId);
+
+		const paymentBody = {
+			transactionID: paymentTxId._toProtobuf(),
+			nodeAccountID: nodeID._toProtobuf(),
+			transactionFee: new Hbar(1).toTinybars(),
+			transactionValidDuration: {
+				seconds: Long.fromInt(120),
+			},
+			cryptoTransfer: {
+				transfers: {
+					accountAmounts:[
+						{
+							accountID: AccountId.fromString(thisAccId)._toProtobuf(),
+							amount: new Hbar(3).negated().toTinybars()
+						},
+						{
+							accountID: nodeID._toProtobuf(),
+							amount: new Hbar(3).toTinybars()
+						}
```

Comment:
Lets add `TODO` here that the exact amount will be computed from `getCost` once `getCost` works without operator